ansible_collections/arista/avd/molecule/eos_config_deploy_cvp/intended/configs/DC1-BL1A.cfg

```diff
@@ -6,6 +6,8 @@ daemon TerminAttr
 !
 vlan internal order ascending range 1006 1199
 !
+ip igmp snooping
```

Comment:
I don't like that we change config on existing deployments. I understand that this _should_ be the EOS default, but I would still prefer this change to be in 4.0.

Reply:
Removed this command.

Reply:
it's back :)